events_test.go

```diff
@@ -52,15 +52,18 @@ func TestEventProcessDisappears(t *testing.T) {
 func TestEventProcessDisappearsDuringDeploy(t *testing.T) {
 	t.Parallel()
 
+	var inits []services.InitSystem
+
 	init := services.MockInit()
-	init.CurrentStatus = services.WithStatus(0, services.Down)
 	act := mockAction()
+	inits = append(inits, init)
+
+	init.CurrentStatus = services.WithStatus(0, services.Down)
 
 	assert.Equal(t, 0, act.Size())
 	svc := &Service{&Entity{"foo", nil, metrics.NewProcessStore("/proc", 15), nil}, act, services.WithStatus(findDownPid(), services.Up), init}
-	svc.Collect(true, func(_ Checkable) {})
```

Comment:
Don't you need to call Collect?  I don't understand what this method is testing anymore.